apps/api/src/domain/models/tag/tag.entity.ts

```diff
@@ -14,7 +14,7 @@ export class Tag extends Resource {
     published = true;
 
```

Comment:
Yes. It will be an `Entity` because it has state that can change over time. This is how we decided to model tags. They have an ID that is referenced by other models and that allows us to update the `text` for a tag without losing the link to all its references. If we decided that tags were just plain text and that renaming a tag would require a global search-and-replace, then a tag would've been a `Value Object`, as opposed to an `Entity`.

A `Resource` on the other hand is a domain concept. It is one of the several special entities that can be referenced in an `Edge Connection` and hence serve as the starting point for a web-of-knowledge graph.